pdns/dnsrecords.hh

```diff
@@ -496,6 +497,31 @@ private:
   string d_keyring;
 };
 
+class SVCBRecordContent : public DNSRecordContent
+{
+public:
+  includeboilerplate(SVCB)
+  const DNSName getTarget() const {return d_target;}
```

Comment:
```suggestion
  const DNSName& getTarget() const {return d_target;}
```
Otherwise it does not make a lot of sense to mark the return value as `const`.